docs/data-management/data-storage-organization.qmd

```diff
@@ -258,9 +258,151 @@ Specifically, they should be human readable, machine readable, and compatible wi
     :::
 
 
-# File Formats
+# File and Data Formats
 
-Guidance coming soon!!!
+## File Formats
+
+The best file format to use will depend on the type and structure of data. In general, data should be stored and/or archived in open formats. Open formats maximize accessibility because they have freely available specifications and generally do not require proprietary software to open them (citation: [UC Santa Barbara's Standard Operating Procedures section on Data Formats](https://emlab-ucsb.github.io/SOP/2.5-data-formats.html)).
```

Comment:
I'm not sure how we want to format the citation. I see a couple other pages use the Quarto `references` feature, but we don't have author names, publication year, etc. for the UCSB website.